ivy_tests/test_ivy/test_frontends/test_numpy/test_random/test_functions.py

```diff
@@ -410,6 +410,40 @@ def test_numpy_standard_gamma(
     )
 
 
+@handle_frontend_test(
+    fn_tree="numpy.random.standard_t",
+    df=st.floats(min_value=1, max_value=20),
```

Comment:
@aparajith21 The tests failed when `df` was so small that it was practically `0` even though the `exclude_min` argument was set to `True`. I changed the `min_value` of `floats` from `0` to `1` and I believe everything passes now 👍

Reply:
Alright, thanks for the contribution! LGTM now then. Sorry for the late review, really busy personally.

Reply:
Please request review again once you resolve merge conflicts, and will then merge it

Reply:
No issues, sorry for the late review, believe it's not directly related to your PR, will go ahead and merge, thanks!